templates/snippets/breadcrumbs.html

```diff
@@ -0,0 +1,15 @@
+{% load static %}
+<nav class="breadcrumbs">
+    <ol>
+        {% for crumb in breadcrumbs %}
+            <li {% if crumb.0 == "Home" %}class="home"{% endif %}>
+            {% if forloop.last %} {# last (current) crumb isn't a link #}
+                <span>{{ crumb.0|truncatechars:20 }}</span>
+            {% else %}
+                <a href="{{ crumb.1 }}">{{ crumb.0|truncatechars:20 }}</a>
+            {% endif %}
+            </li>
+        {% endfor %}
+    </ol>
+    <script type="application/ld+json">{{ rdf|safe }}</script>
```

Comment:
Is this the right place to include the rdf? Would it be better at the end of the document?  I don't know if there are any conventions for where it should go.

It doesn't seem like something that belongs in the breadcrumb template. Maybe it could just be included in an appropriate base page template and output if set. We should probably use a slightly more descriptive/specific template variable name in any case, but especially if we do that.  Maybe `page_jsonld` or something similar? `schema_jsonld` ?

Reply:
according to google it can go pretty much anywhere in the `<body>` or even in the `<head>`, so I'll move it inside the `{% block js %}` and out of the breadcrumb snippet.